src/database/migrations/1739918380436_member_user_public_key.ts

```diff
@@ -0,0 +1,23 @@
+import { type Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+    await db.schema
+        .createTable('member_user_public_key')
+        .addColumn('id', 'uuid', (col) => col.defaultTo(sql`v7uuid()`).primaryKey())
+        .addColumn('user_id', 'uuid', (col) => col.notNull())
+        .addColumn('value', 'text', (col) => col.notNull())
+        .addColumn('fingerprint', 'text', (col) => col.notNull())
+        .addColumn('created_at', 'timestamp', (col) => col.defaultTo(sql`now()`).notNull())
+        .addColumn('updated_at', 'timestamp', (col) => col.defaultTo(sql`now()`).notNull())
+        .execute()
+
+    db.schema
+        .createIndex('member_user_public_key_fingerpring_indx')
```

Comment:
I added this index with the expectation we're going to query by fingerprint. Perhaps that's premature optimization and / or other indexes might be useful?